test/integration/dockerapi_fetcher_test.go

```diff
@@ -4,15 +4,16 @@ import (
 	"context"
 	"encoding/json"
 	"testing"
+	"time"
 
 	"github.com/docker/docker/client"
 	"github.com/newrelic/nri-docker/src/raw/dockerapi"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-// TODO: this should extended/replaced when the fetcher actually fetches something.
 // It currently it shows how to get container stats data and can be useful for development purposes.
-func TestDockerAPIHelpers(t *testing.T) {
+func TestDockerAPIFetcher(t *testing.T) {
```

Comment:
Would it make sense to assure somehow we are executing the test in a system with cgroups-v2 to avoid false-positives?

I guess we won't face that issue in the CI. But, at least, I'd add a comment explaining the requirement.

Reply:
make sense!

Reply:
I added a check, as we mention if we need this check in other tests here it might be good to add a suite and checkit once